### DISCUSSION TITLE
Restrict users to only access data of specified tenants

### DISCUSSION THREAD

**Discussion by joyhu** [netbox-community/netbox#15395] [General]
We are a company that does network maintenance for different customers. We hope to enter all customer data into netbox. Different customers are represented by tenants in netbox, but we hope that company employees can only see the data of authorized tenants when accessing netbox. . Is this possible in the current version?

**Comment by Mailstorm-ctrl** [comment]
You can try custom permission constraints. It will take some trial and error as the tenant field isn't available everywhere. I'm not sure if you could use tags to achieve this as those are available basically everywhere.

https://demo.netbox.dev/static/docs/administration/permissions/#constraints

**Reply by joyhu** [reply]
thank you!
